init.sls

```diff
@@ -104,22 +104,45 @@ chown_pgdata:
       - pkg: postgresql
 
 # Deploy users. Sort them, so order is not changing between salt runs
-{% for config in pillar['postgresql']['users']|default({}) %}
+# Reverse the order for more convenient ownership of databases in multiuser scenarios
```

Comment:
I think there should be a not on what's meat by this. Something like "Because `postgres_user.present` automatically promotes the last user as the database owner, we're reversing the order. This ensures backwards compatibility to previous configurations and also is more intuitive" - or something like that.

Reply:
What about:
Reverse the order so the first user defined in the pillar is database owner (as in previous versions), since postgres_database.present updates ownership in every iteration

Reply:
perfect!